app/src/main/res/layout/fragment_contributions.xml

```diff
@@ -7,6 +7,15 @@
     android:background="?attr/mainBackground"
     >
 
+    <TextView
+        android:id="@+id/noDataYet"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/no_uploads"
+        android:gravity="center"
+        android:layout_centerInParent="true"
```

Comment:
Add left and right margins please :)

Reply:
Done
![screenshot_20181019-111508](https://user-images.githubusercontent.com/43948534/47209469-b7f77780-d390-11e8-8880-c5ed54d02992.jpg)